FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -750,15 +1096,15 @@ components:
           type: string
           format: date-time
           description: The last timestamp when this message's information was guaranteed to be accurate. If this field is not provided, it is assumed that the time of receipt of this message is the time of validity
-        systemTimeout:
+        timeout:
           type: integer
           minimum: 1
           example: 100
-          description: Seconds within which a system update is expected from the client. If no update is received, the system will be considered lifeless.
+          description: Seconds within which a component update is expected from the client. If no update is received, the system will consider the component lifeless.
```

Comment:
By system, you probably mean FRAIHMWORK.

Reply:
Addressed